### PR TITLE
PS-777 Resolve configuration variables

### PR DESCRIPTION
**Pull request by pivnicek**
uf, konecne.

@JakubMatejka  this one may seem a little complicated, but basically the src change is to grab the configuration details from docker-runner rather than from sapi so that DR can do variable replacement.

### CODE REVIEW

File: src/ConfigValidator.php [hunk 1]
```diff
@@ -174,6 +201,7 @@ private function validateTransformationV2Config(): void
                 $scriptLength += strlen(implode("\n", $codeChunk['script']));
             }
         }
+        var_dump($this->codeChunks);
```
Comment (JakubMatejka) [inline src/ConfigValidator.php]: 🚫 😄

Comment (pivnicek) [reply src/ConfigValidator.php]: lmao thank you!

Comment (pivnicek) [reply src/ConfigValidator.php]: oh, there's 2 of htem!

File: tests/functional/TransformationV2SandboxTest.php [hunk 2]
```diff
@@ -142,4 +141,123 @@ public function testFullTransformationV2Config(): void
...
+                    ],
+                ],
+                'variables_id' => $variablesId,
+                'variables_values_id' => $variableValuesId,
+                'shared_code_id' => $sharedCodeId,
+                'shared_code_row_ids' => [$sharedCodeRowId]
+            ]
+        );
+        $createdConfig = $components->addConfiguration($configuration);
+        var_dump($createdConfig);
```
Comment (JakubMatejka) [inline tests/functional/TransformationV2SandboxTest.php]: dtto

File: src/ConfigValidator.php [hunk 3]
```diff
@@ -158,6 +184,7 @@ private function validateTransformationV2Config(): void
                 new TransformationV2Config(),
                 ['configuration' => $configData['configuration']]
             );
+            var_dump($configData);
```
Comment (JakubMatejka) [inline src/ConfigValidator.php]: and other one

**Comment by pivnicek** [comment]
sorry, should have waited for the green light from travis.  🤞

**Review by JakubMatejka — APPROVED** [review]
Looks ok

**Comment by JakubMatejka** [comment]
(once the Travis is green 😄 )

**Comment by pivnicek** [comment]
jasne;). diky moc!

**Comment by pivnicek** [comment]
Ah, locally I use my 580 for testing and that user has a user-defined template so that was on the expected output notebook (which is cool to see that working too :).  but the test user on travis doesn't have it

**Comment by pivnicek** [comment]
tak, green:)

**Pull request merged**